packages/cli/src/environments.ee/source-control/source-control-export.service.ee.ts

```diff
@@ -211,7 +211,12 @@ export class SourceControlExportService {
 				};
 			}
 			const fileName = getVariablesPath(this.gitFolder);
-			const sanitizedVariables = variables.map((e) => ({ ...e, value: '' }));
+			const sanitizedVariables = variables.map((e) => ({
```

Comment:
```suggestion
			const sanitizedVariables: ExportableVariable[] = variables.map((e) => ({
```

This would ensure that the variable saved in `variable_stubs.json` have the same shape as the one in `{projectId}.json`